src/main.rs

```diff
@@ -108,40 +115,78 @@ async fn cmd_remove(args: RemoveArgs, options: Options) {
 async fn cmd_search(args: SearchArgs, options: Options) {
     let query_string = args.search;
 
-    if args.aur {
-        tracing::info!("Searching AUR for {}", &query_string);
-        operations::aur_search(&query_string, args.by, options).await;
-    }
-    if args.repo {
+    let mut results = Vec::new();
+
+    let arg1 = env::args().collect::<Vec<String>>()[1].clone();
```

Comment:
Can we express this with clap?

Reply:
The way it's currently implemented, clap sees it as a hidden alias, we just parse it before actually passing anything down to further functions.

I'm not sure if there's a way to handle this with clap as `-Q` is a subcommand, not a flag.

Reply:
Hmm, let's leave it as it is and open an enhancement issue?